tooling/nargo_cli/Cargo.toml

```diff
@@ -91,7 +91,7 @@ iai = "0.1.1"
 test-binary = "3.0.2"
 test-case.workspace = true
 lazy_static.workspace = true
-light-poseidon = "0.2.0"
+light-poseidon = { git = "https://github.com/Lightprotocol/light-poseidon", rev = "c3bb062" }
```

Comment:
Looks like this dependency is completely unused so we can just get rid of it! (we ignore unused dev-deps due to false positives)

Reply:
Ah, nevermind. I searched for `light-poseidon` but forgot that `-` would get changed to `_` in the actual source.

Reply:
This is just in the test suite so we can tolerate the performance degradation from needing to convert to ark0.4 fields however.